plugins/menubar/linux/src/menubar_plugin.cc

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <menubar/menubar_plugin.h>
+
+#include <gtk/gtk.h>
+#include <stdio.h>
+
+#include "../../common/channel_constants.h"
+
+static constexpr char kWindowTitle[] = "Flutter Menubar";
+
+namespace plugins_menubar {
+using flutter_desktop_embedding::JsonMethodCall;
+using flutter_desktop_embedding::MethodResult;
+
+MenubarPlugin::MenubarPlugin()
+    : JsonPlugin(kChannelName, false), menubar_(nullptr) {}
+
+MenubarPlugin::~MenubarPlugin() {}
+
+// Class containing the implementation of the Menubar widget. This is currently
+// a floating GTK window, separate from the main app. This is not the optimal
+// solution.
+class MenubarPlugin::Menubar {
+ public:
+  explicit Menubar(MenubarPlugin *parent) {
+    menubar_window_ = gtk_window_new(GTK_WINDOW_TOPLEVEL);
+    gtk_window_set_position(GTK_WINDOW(menubar_window_), GTK_WIN_POS_CENTER);
+    gtk_window_set_default_size(GTK_WINDOW(menubar_window_), 300, 50);
+    gtk_window_set_title(GTK_WINDOW(menubar_window_), kWindowTitle);
+
+    GtkWidget *vbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 0);
+    gtk_container_add(GTK_CONTAINER(menubar_window_), vbox);
+
+    menubar_ = gtk_menu_bar_new();
+    gtk_box_pack_start(GTK_BOX(vbox), menubar_, FALSE, FALSE, 0);
+  }
+  virtual ~Menubar() {
+    if (menubar_window_) {
+      gtk_widget_destroy(menubar_window_);
+      menubar_window_ = nullptr;
+    }
+  }
+
+  // Get the top level menubar widget.
```

Comment:
s/Get/Gets/

Per Google C++ style guide on function comment form. Same with the other function comments.